lib/apollo-client.ts

```diff
@@ -45,7 +45,7 @@ const getGraphqlUrl = (apiVersion, internal = false) => {
 };
 
 const getCustomAgent = () => {
-  if (!customAgent) {
+  if (typeof window === 'undefined' && !customAgent) {
```

Comment:
Would testing `process.browser` work the same?

Reply:
I think so, but `process.browser` is giving a deprecated warning and pointing to use the window check

Reply:
Good to know!